.travis.yml

```diff
@@ -1,6 +1,39 @@
 sudo: false
 language: node_js
-
 node_js:
-  - 0.10
-  - 4
+    - "0.10"
+    - "0.12"
+    - "4"
+    - "iojs"
```

Comment:
Sorry, that example was just a guide. Hapi only supports versions of Node, not io. So please remove them from here and the exclusion matrix. Node versions should be 0.10, 0.12,4, and 4.0. You'll need to update the exclusion matrix as well.